tiny_dnn/core/kernels/conv2d_op_avx.h

```diff
@@ -256,7 +254,7 @@ void avx_conv2d_5x5_kernel(const core::conv_params& params,
 
     if (out.height_ == 1 && out.width_ == 1) {
         const double* pw = &W[0];
-        for (size_t o = 0; o < out.depth_; ++o) {
+        for (serial_size_t o = 0; o < out.depth_; ++o) {
```

Comment:
Because the type of `tiny_dnn::core::conv_params::out::depth_` is `serial_size_t`.
The type of `tiny_dnn::core::conv_params::out` is `serial_size_t`.
I've already forgotten why nyanp started using `serial_size_t` with `cereal` library though he mentioned the reason several times in the past.